RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -104,54 +104,80 @@ namespace SDL {
   constexpr float PT_CUT = 0.8f;
 #endif
 
-  const unsigned int MAX_BLOCKS = 80;
-  const unsigned int MAX_CONNECTED_MODULES = 40;
+  constexpr unsigned int MAX_BLOCKS = 80;
+  constexpr unsigned int MAX_CONNECTED_MODULES = 40;
 
-  const unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
+  constexpr unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
 
-  const unsigned int N_MAX_PIXEL_MD_PER_MODULES = 2 * N_MAX_PIXEL_SEGMENTS_PER_MODULE;
+  constexpr unsigned int N_MAX_PIXEL_MD_PER_MODULES = 2 * N_MAX_PIXEL_SEGMENTS_PER_MODULE;
 
-  const unsigned int N_MAX_PIXEL_TRIPLETS = 5000;
-  const unsigned int N_MAX_PIXEL_QUINTUPLETS = 15000;
+  constexpr unsigned int N_MAX_PIXEL_TRIPLETS = 5000;
+  constexpr unsigned int N_MAX_PIXEL_QUINTUPLETS = 15000;
 
-  const unsigned int N_MAX_PIXEL_TRACK_CANDIDATES = 30000;
-  const unsigned int N_MAX_NONPIXEL_TRACK_CANDIDATES = 1000;
+  constexpr unsigned int N_MAX_PIXEL_TRACK_CANDIDATES = 30000;
+  constexpr unsigned int N_MAX_NONPIXEL_TRACK_CANDIDATES = 1000;
 
-  const unsigned int size_superbins = 45000;
+  constexpr unsigned int size_superbins = 45000;
 
   //defining the constant host device variables right up here
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleBarrel[6] = {0.0052, 0.0038, 0.0034, 0.0034, 0.0032, 0.0034};
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleEndcap[5] = {0.006, 0.006, 0.006, 0.006, 0.006};
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniRminMeanBarrel[6] = {
+  namespace objLayers {
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpLS =
+        2;  // Currently pixel tracks treated as LSs with 2 double layers (IT layers 1+2 and 3+4). To be potentially handled better in the future.
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kLS = 2;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kT3 = 3;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpT3 = 5;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kT5 = 5;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpT5 = 7;
+  }  // namespace objLayers
+
+  namespace objHits {
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpLS =
+        4;  // Currently pixel tracks treated as LSs with 4 hits. To be potentially handled better in the future.
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kLS = 4;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kT3 = 6;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpT3 = 10;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kT5 = 10;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpT5 = 14;
+  }  // namespace objHits
+
+  ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMulsInGev2 =
+      0.015;  // 15 MeV constant from the approximate Bethe-Bloch formula
```

Comment:
please keep longer comments above the line of code; the clang-formatting breaks the lines in a weird way

Reply:
BTW, why `Gev2`? either the value should be `0.015 * 0.015` or it should be `GeV` (also `V`)